docs/application/dotnet/guides/wcircularui/quickstart.md

```diff
@@ -0,0 +1,145 @@
+# Quickstart
+
+This guide explains how to create an application using **Toast** and **CircleSlider** UI contols. 
+
+- When you press the button, **Toast** pops up and it automatically disappears when the timer timeout (in seconds).
+- When you rotate the bezel of Tizen wearable device, **CircleSlider** bar moves either in forward or in backward direction.
+
+## Prerequisite
+
+- Visual Studio
+- [Visual Studio tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen)
+
+## Set Up Development Environment and Create Project
+
+Following are the steps to create a Tizen Wearable App Template:
+
+- [Create Tizen Xamarin.Forms](#create-tizen-xamarin-forms-application)
+- [Create Project](#create-project)
+- [Insert CircularUI Control Code](#insert-circularui-control-code)
+- [Build and Launch Your Application](#build-and-launch-your-application)
+
+### Create Tizen Xamarin Forms Application
+
+Create a Tizen Xamarin Forms application. For more information, see [Creating Your First Tizen .NET Application](https://developer.tizen.org/development/training/.net-application/creating-your-first-tizen-.net-application).
+
+In this guide, project name is `SampleCircleApp`.
+
+### Create Project
+
+In **New project** menu, select **Tizen** > **Tizen Wearable App**. Click **OK**.
+    ![tizen_project_wizard_capture_template](media/tizen_project_wizard_capture_template.png)
+
+You can use APIs of Xamarin.Forms and Tizen.Wearable.CircularUI now.
+    ![tizen_project_wizard_capture_template2](media/tizen_project_wizard_capture_template2.png)
+
+   > **Note**
+   >
+   > See [Tizen Wearable XAML App template](quickstart_tizenxamlapptemplate.md), if you want to choose **Tizen Wearable XAML App template**.
```

Comment:
@divyk, isn't it encouraged to describe the condition first, and the required action later in the developer document?
As like in "For more **Tizen Wearable XAML App template** information , see [guide of Tizen XAML App template](quickstart_tizenxamlapptemplate.md)" form.

Reply:
I apply below sentence.

   > For more **Tizen Wearable XAML App template** information, see [Tizen Wearable XAML App template](quickstart_tizenxamlapptemplate.md).